frontend/src/main/components/UCSBSubjects/UCSBSubjectsTable.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+// import { toast } from "react-toastify";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/UCSBSubjectUtils"
+import { useNavigate } from "react-router-dom";
+import { hasRole } from "main/utils/currentUser";
+
+export default function UCSBSubjectsTable({ subjects, currentUser }) {
+
+    const navigate = useNavigate();
+
+    const editCallback = (cell) => {
+        navigate(`/ucsbsubjects/edit/${cell.row.values.id}`)
```

Comment:
This is the API mismatch I am talking about. Controllers uses `/UCSBSubjects` and frontend table uses `/ucsbsubjects`
![image](https://user-images.githubusercontent.com/49248229/157352419-e7f38712-f148-4d1b-8fc2-526504a77acd.png)